packages/RichTextEditing-Core.package/RichTextStructureEditor.class/instance/isActiveAttributeAt..st

```diff
@@ -0,0 +1,4 @@
+text attributes
+isActiveAttributeAt: anIndex
+
+	^ self selectedStructure includes: (TextStructure new: (self attributeSymbolAt: anIndex))
```

Comment:
Where is the difference between `selectedStructure` and `currentStructure`?

Reply:
There is none. I started renaming currentStructure to selectedStructure but then realized it was pointless.